<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLiveryForm</class>
 <widget class="QFrame" name="CLiveryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Livery editor</string>
  </property>
  <layout class="QVBoxLayout" name="vl_LiveryForm">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Livery">
     <property name="title">
      <string>Livery</string>
     </property>
     <layout class="QGridLayout" name="gl_Livery">
      <item row="0" column="0">
       <widget class="BlackGui::CLabelAndIcon" name="lai_Id">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="wi_DesignatorVa" native="true">
        <layout class="QHBoxLayout" name="hl_DesignatorRank">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="BlackGui::Components::CDbLiverySelectorComponent" name="livery_Selector">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="wi_Colors" native="true">
        <layout class="QHBoxLayout" name="hl_Colors">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="BlackGui::CColorSelector" name="color_Fuselage">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="BlackGui::CColorSelector" name="color_Tail">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="BlackGui::Editors::CAirlineIcaoForm" name="editor_AirlineIcao">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>225</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_Code">
        <property name="text">
         <string>Code:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_Description">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>le_Description</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_Timestamp">
        <property name="text">
         <string>Last updated:</string>
        </property>
        <property name="buddy">
         <cstring>le_Updated</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_Updated">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_Id">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_FuselageColor">
        <property name="text">
         <string>Fuselage/Tail:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_Description">
        <property name="placeholderText">
         <string>Livery description</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="BlackGui::CDropSite" name="drop_DropData">
        <property name="text">
         <string>Drop data here</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="7">
       <widget class="BlackGui::Editors::CValidationIndicator" name="val_Indicator">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CDropSite</class>
   <extends>QLabel</extends>
   <header>blackgui/dropsite.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::CLabelAndIcon</class>
   <extends>QFrame</extends>
   <header>blackgui/labelandicon.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CValidationIndicator</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/validationindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CColorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/colorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CAirlineIcaoForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/airlineicaoform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbLiverySelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbliveryselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Id</tabstop>
  <tabstop>le_Description</tabstop>
  <tabstop>le_Updated</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
